Admin/AbstractAdmin.php

```diff
@@ -2807,52 +2845,15 @@ protected function configureRoutes(RouteCollection $collection)
     /**
      * Configure buttons for an action.
      *
+     * @param array  $buttonList
      * @param string $action
      * @param object $object
      *
      * @return array
      */
-    protected function configureActionButtons($action, $object = null)
+    protected function configureActionButtons($buttonList, $action, $object = null)
```

Comment:
I think the problem is reversed.

The `$action` should be renamed to `$currentAction` as this is the current one, isn't it?

And then, we can keep have `$actionButtons`.

What do you think?